### ISSUE TITLE
Seg fault with arrays

### ISSUE THREAD

**Issue by jjaraalm** [epics-base/pvaPy#71]
Just installed pvapy 4.0.3 and epics-base 7.0.6 from the epics conda channel with python 3.8. Existing code that was working with previous pvapy versions now fails, and I've traced it back to the following minimal example:

```python
import pvaccess as pva
pva.PvObject({'foo': [pva.STRING]}) # Raises seg. fault
```

This is not limited to `pva.STRING` types, but also occurs with `pva.INT` and possibly others as well. Any suggestions? Otherwise I'll have to roll back.

**Comment by jjaraalm** [comment]
Actually, maybe this is an issue with the installed epics-base and not pvapy directly.

**Comment by jjaraalm** [comment]
As an update: the same version installed from pip works fine, so I'm using that for now.

**Comment by sveseli** [comment]
I just tested conda 3.8 package, and it works okay:

<pre>
$ conda list pvapy
# packages in environment at /local/sveseli/PVAPY/RELEASE-3.8-TEST/CONDA:
#
# Name                    Version                   Build  Channel
pvapy                     4.0.3                    py38_1    epics
pvapy-boost               1.72.0                   py38_1    epics
$ python -c "import pvaccess as pva; pva.PvObject({'foo': [pva.STRING]})"
$ python -c "import pvaccess as pva; print(pva.PvObject({'foo': [pva.STRING]}))"
structure 
    string[] foo []
</pre>

Could you please list numpy package in your conda installation that results in segfault?

<pre>
$ conda list numpy
# packages in environment at /local/sveseli/PVAPY/RELEASE-3.8-TEST/CONDA:
#
# Name                    Version                   Build  Channel
numpy                     1.20.3           py38hf144106_0  
numpy-base                1.20.3           py38h74d4b33_0  
</pre>

**Comment by sveseli** [comment]
I believe these segfault issues have been resolved as of numpy 1.22.
